.travis.yml

```
@@ -7,32 +9,29 @@ stages:
 
 if: branch = master OR type = pull_request
 
-stage: test
-
-name: Project Tests
-
-language: python
-
-python:
-  - "3.8"
-  - "3.9"
-
-install: skip
-
-script:
-  - |
-    set -e
-    make docker/run target=python/test
-    make localstack/up
-    make localstack/pytest
-    make localstack/clean
+before_install:
+  - tmpdaemon=$(mktemp)
+  - sudo jq '."registry-mirrors" += ["https://mirror.gcr.io"]' /etc/docker/daemon.json > "$tmpdaemon"
+  - sudo mv "$tmpdaemon" /etc/docker/daemon.json
+  - sudo systemctl daemon-reload
+  - sudo systemctl restart docker
+  - docker system info
 
 jobs:
   include:
     - stage: lint
       name: Project Syntax Verification
-      script:
-        - make docker/run target=lint
+      script: make docker/run target=lint
+    - stage: test
+      name: Run unit tests
+      script: make docker/run target=pytest/lambda/tests
+    -
```

I'd really prefer to be explicit about the stage, rather than used the implied default value. (It's not picking it up from the prior item in the list, it's getting `test` as the travis-ci default value for the `stage` argument.)

```suggestion
    - stage: test
```

---

ok